frontend/src/App.js

```diff
@@ -378,6 +379,25 @@ export default function App() {
                   component={() => <RoutineReports />}
                   role="Reports"
                 />
+              <SecureRoute
+                  path="/ByTestType"
+                  exact
+                  component={() => <RoutineIndex />}
+                  role="Reports"
+                />
+                <SecureRoute
+                  path="/ByPanelType"
+                  exact
+                  component={() => <RoutineIndex />}
+                  role="Reports"
+                />
+                <SecureRoute
+                  path="/ByUnit"
```

Comment:
I also think you don't need to register this route here and these others you have registered aswell since `ByTestType`, `ByPanelType` and `ByUnit` are really under the `/RoutineReport` url.

Reply:
After implementing your changes,  it is kinda not direct while redirecting through those pages, so i would think keeping it like that would even make the code to be understable quicker and explicit

Reply:
Thanks @mherman22 
@sherrif10 can you only use the existing route `/RoutineReport` 
you dont need the other routes